homeassistant/components/zwave/__init__.py

```diff
@@ -221,22 +221,12 @@ def setup(hass, config):
     descriptions = conf_util.load_yaml_config_file(
         os.path.join(os.path.dirname(__file__), 'services.yaml'))
 
-    try:
-        import libopenzwave
-    except ImportError:
-        _LOGGER.error("You are missing required dependency Python Open "
-                      "Z-Wave. Please follow instructions at: "
-                      "https://home-assistant.io/components/zwave/")
-        return False
     from pydispatch import dispatcher
     # pylint: disable=import-error
     from openzwave.option import ZWaveOption
     from openzwave.network import ZWaveNetwork
     from openzwave.group import ZWaveGroup
 
-    default_zwave_config_path = os.path.join(os.path.dirname(
```

Comment:
Is the configuration not a part of the package? That is unfortunate. I wonder if we should auto download the config in a future PR. Let's make sure that we note in the docs how to obtain it.